### DISCUSSION TITLE
Highlighting bash

### DISCUSSION THREAD

**Discussion by PadowYT2** [vuejs/vitepress#3446] [Q&A] [answered]
Hello, why does the `d` in this example doesn't highlight?
![https://cdn.padow.ru/jKsNM](https://cdn.padow.ru/jKsNM)

**Comment by brc-dd** [comment, marked as answer]
Ah, it's same as what VSCode does: (mainly because it's not able to parse that syntax as it's not valid?)

<img width="295" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/3923c0c7-5302-4bb8-8834-1848c298c516">

**Reply by PadowYT2** [reply]
Alright, thank you. Not sure if this is a problem with highlight.js or just as intended?

**Reply by brc-dd** [reply]
We aren't using highlight.js. Highlight.js seems to be parsing it correctly. But this is mainly garbage in, garbage out. So, not sure if VSCode team will be accepting a fix (also that grammar is now pointing to an archived repo - https://github.com/microsoft/vscode/blob/main/extensions/shellscript/syntaxes/shell-unix-bash.tmLanguage.json to https://github.com/jeff-hykin/better-shell-syntax/blob/master/autogenerated/shell.tmLanguage.json to https://github.com/atom/language-shellscript/blob/master/grammars/shell-unix-bash.cson).

**Reply by brc-dd** [reply]
BTW you can do something like this if you want to escape highlighting for that part:

````md
```sh-vue
php artisan help {{'<command>'}}
```
````

<img width="289" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/5f137d43-5d14-4acb-9506-fe95cc3be278">
